R/plumber-step.R

```diff
@@ -102,21 +105,6 @@ PlumberStep <- R6Class(
 
 # @param positional list with names where they were provided.
 getRelevantArgs <- function(args, plumberExpression) {
-  if (length(args) == 0) {
-    unnamedArgs <- NULL
-  } else if (is.null(names(args))) {
-    unnamedArgs <- 1:length(args)
-  } else {
-    unnamedArgs <- which(names(args) == "")
-  }
-
-  if (length(unnamedArgs) > 0 ) {
-    stop("Can't call a Plumber function with unnammed arguments. Missing names for argument(s) #",
-         paste0(unnamedArgs, collapse=", "),
-         ". Names of argument list was: \"",
-         paste0(names(args), collapse=","), "\"")
-  }
-
   # Extract the names of the arguments this function supports.
   fargs <- names(formals(eval(plumberExpression)))
```

Comment:
`eval()` can probably be removed. If `plumberExpression` is not a function at this point we already have problems.

Reply:
If not removed, definitely needs an explicit evaluation environment.